### PR TITLE
Better disabled cards and colours/names

### PR DESCRIPTION
**Pull request by tomjn**
More colours, more names, and it now tells you who hasn't voted yet:

Screenshot 2024-02-09 at 14 39 39

Also it's clearer about when you're a spectator:

Screenshot 2024-02-09 at 14 43 00

Also includes some easter eggs:

Screenshot 2024-02-09 at 14 15 40

### CODE REVIEW

File: server/room.js [hunk 1]
```diff
@@ -80,13 +80,67 @@ Room.prototype.enter = function(socket, data) {
...
+        'captain janeway',
+        'commander sisko',
+        'millenium puzzle',
+        'scrum master',
+        'director',
+        'bindelstick',
+        'pingu',
+        'dobby',
+        'yugi',
+        'harry seldon',
```
Comment (rmccue) [inline server/room.js:141]:
```suggestion
        'hari seldon',
```

**Review by rmccue — DISMISSED** [review]
Presume you're not sticking to the HM Coding Standards here; mostly looks good from a pure functionality standpoint.

**Comment by tomjn** [comment]
@rmccue yup, I'd rather rebuild it in something other than Angular than spend a tonne of time redoing to standards, Hari's been fixed up

File: server/room.js [hunk 2]
```diff
@@ -1,5 +1,118 @@
...
+  'dobby',
+  'droid',
+  'engineer',
+  'exodia',
+  'geordi laforge',
+  'han solo',
+  'hari seldon',
+  'hypersphere',
+  'jedi',
+  'luxanna troi',
```
Comment (Sephsekla) [inline server/room.js:92]:
```suggestion
  'lwaxana troi',
```

File: server/room.js [hunk 3]
```diff
@@ -1,5 +1,118 @@
...
+  'mandalorian',
+  'millenium puzzle',
+  'oddish',
+  'pikachu',
+  'pingu',
+  'plant pot',
+  'poet',
+  'riker',
```
Comment (Sephsekla) [inline server/room.js:100]:
```suggestion
  'riker',
  'riker transporter clone (thomas)',
```

File: server/room.js (hunk 2 above)
Comment (tomjn) [reply server/room.js:92]: nice catch

File: server/room.js (hunk 3 above)
Comment (tomjn) [reply server/room.js:100]: I appreciate this but the card is only so big, I will however add this as a consolation:

```suggestion
  'riker',
  'transporter clone',
  'spock',
```

**Review by Sephsekla — APPROVED** [review]

**Pull request merged**